pwintools.py

```diff
@@ -642,26 +649,26 @@ def send(self, buf):
         
     def sendline(self, line):
         """sendline(line) sends the line adding newline to the process stdin"""

```

Comment:
If would be nice to support both bytes and str in sendline.
Something like:
```python
        if isinstance(line, (bytes, bytearray)):
            self.write(line + self.newline)
        else:
            self.write(line + "\n")
```

Reply:
No. In python3 str has a different signature than bytes. Bytes alway have the same form. It is the only way to exploit buffer overflow. powntools already use bytes